host-bmc/dbus/custom_dbus.cpp

```diff
@@ -160,20 +163,20 @@ void CustomDBus::implementVRMInterface(const std::string& path)
 {
     if (vrm.find(path) == vrm.end())
     {
-        vrm.emplace(
-            path, std::make_unique<ItemVRM>(pldm::utils::DBusHandler::getBus(),
-                                            path.c_str()));
+        vrm.emplace(path,
+                    std::make_unique<VRM>(pldm::utils::DBusHandler::getBus(),
+                                          path.c_str()));
     }
 }
 
-void CustomDBus::implementCpuCoreInterface(const std::string& path)
+void CustomDBus::implementCpuCoreInterface(const std::string& path, bool value)
 {
     if (cpuCore.find(path) == cpuCore.end())
     {
         cpuCore.emplace(
-            path, std::make_unique<CoreIntf>(pldm::utils::DBusHandler::getBus(),
-                                             path.c_str()));
-        implementObjectEnableIface(path);
+            path, std::make_unique<CPUCore>(pldm::utils::DBusHandler::getBus(),
+                                            path.c_str()));
+        implementObjectEnableIface(path, value);
     }
```

Comment:
remove this from here & call this from host_pdr_handler.cpp (if entity type is core)

Reply:
> remove this from here & call this from host_pdr_handler.cpp (if entity type is core)

Done